frontend/public/components/secret.jsx

```diff
@@ -115,7 +115,11 @@ const SecretsPage = props => {
     createLink: (type) => `/k8s/ns/${props.namespace}/secrets/new/${type !== 'yaml' ? type : ''}`
   };
 
-  return <ListPage ListComponent={SecretsList} canCreate={true} rowFilters={filters} createButtonText="Create" createProps={createProps} {...props} />;
+  // Only show row filters if no props.filters were passed. We want to hide the
+  // filter bar when already filtering secrets for pages like the service
+  // account page.
+  const rf = _.isEmpty(props.filters) ? rowFilters : null;
```

Comment:
Debating if this change shouldn't be made in ListPage

/hold

Reply:
I think moving it down to the ListPage makes sense.